shared/rakefiles/ci.rake

```diff
@@ -5,6 +5,10 @@ task :set_vars_ci => [:set_vars] do
     " run ",
```

Comment:
@amatas Maybe I am missing something, but it is not clear to me why do we need another `shared` dir just for rake files? Our shared Helm charts live in `common/charts`, why don't have rake files in `common/rakefiles`?

Reply:
We need a separate directory outside the `common` and `gcp` directory where [these imports of the entrypoint.rake](https://github.com/gpii-ops/gpii-infra/blob/d2a4efb99a5dd9c02806d3ac16f92ff7f43244be/shared/rakefiles/entrypoint.rake#L3-L4) could work in both cases because it uses the same relative path. If the rake files live in the `common` directory the import would be different if you call the rakes from `common` and `gcp`, so in one of the cases will fail.

Reply:
@amatas If it is only about relative path for rake file imports, then this should do the trick:
```
@rakefiles_dir = File.dirname(__FILE__)

import "#{@rakefiles_dir}/ci.rake"
import "#{@rakefiles_dir}/test.rake"
```
I would be very happy if we get rid of additional `shared` dir and let our rake files live in `common`!

Reply:
@natarajaya could we move forward this PR and tackle this in a separate ticket?. This change was discussed at the beginning with @mrtyler (https://github.com/gpii-ops/gpii-infra/pull/60/commits/69a69b2cdb7d6f2f34c0ee829ff28a767c07a873) , and we found the solution of using a shared directory for shared resources like helm charts and rakefiles that are not tied directly with a particular environment (common is an environment itself with a particular pipeline). I wouldn't like to revert this work now.

Seriously, if the agreement is to have the rakefiles inside `common` I will be fine with that. But please, let's merge this PR first and revert this change if so later.